src/contractTest/java/uk/gov/hmcts/reform/orgrolemapping/OrgRoleMappingConsumerTest.java

```diff
@@ -93,6 +151,96 @@ void getListOfRolesAndGet200Test(MockServer mockServer)
         assertThat(role.get("category"), equalTo("JUDICIAL"));
     }
 
+    @Test
+    @PactTestFor(pactMethod = "executeCreateRoleAssignmentReplacingExistingFalseAndGet201")
+    void createRoleAssignmentReplaceExistingFalseAndGet201Test(MockServer mockServer)
+            throws JSONException {
+        String actualResponseBody =
+                SerenityRest
+                        .given()
+                        .headers(getHttpHeaders())
+                        .contentType(ContentType.JSON)
+                        .body(createRoleAssignmentRequestReplaceExistingFalse())
+                        .post(mockServer.getUrl() + RAS_CREATE_ROLE_ASSIGNMENT_URL)
+                        .then()
+                        .log().all().extract().asString();
+
+        JSONObject response =  new JSONObject(actualResponseBody);
+        Assertions.assertThat(response).isNotNull();
+
+        JSONObject roleRequest = response.getJSONObject("roleAssignmentResponse").getJSONObject("roleRequest");
+        JSONArray requestedRoles = response.getJSONObject("roleAssignmentResponse").getJSONArray("requestedRoles");
+        JSONObject role1 = requestedRoles.getJSONObject(0);
+        assertThat(roleRequest.get("status"), equalTo("APPROVED"));
+        assertThat(roleRequest.get("requestType"), equalTo("CREATE"));
+        assertThat(roleRequest.get("process"), equalTo(roleRequest.get("reference")));
```

Comment:
It is not mandatory to have both process and reference equal.
We can assert them separately.
assertThat(roleRequest.get("process"), equalTo("S-028"));
assertThat(roleRequest.get("reference"), equalTo("S-028"));

Reply:
Done